lib/src/blocs/forgot_password_bloc/forgot_password_state.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:equatable/equatable.dart';
+
+abstract class ForgotPasswordState extends Equatable {
+  final String email;
+  final String emailErrorMessage;
+  final String unknownErrorMessage;
+  const ForgotPasswordState(
+      {this.email = '',
+      this.emailErrorMessage = '',
+      this.unknownErrorMessage = ''});
+
+  @override
+  List<Object> get props => [email, emailErrorMessage, unknownErrorMessage];
+}
+
+class ForgotPasswordInitial extends ForgotPasswordState {}
+
+class ForgotPasswordInProgress extends ForgotPasswordState {}
+
+class ForgotPasswordSuccess extends ForgotPasswordState {}
+
+class ForgotPassworFailure extends ForgotPasswordState {
+  ForgotPassworFailure({
+    String emailErrorMessage = '',
+    String unknownErrorMessage = '',
+  }) : super(
+          emailErrorMessage: emailErrorMessage,
+          unknownErrorMessage: unknownErrorMessage,
+        );
```

Comment:
why did you use super here? 
where is my   @override
  List<Object> get props =>